rosdep/python.yaml

```diff
@@ -2771,6 +2771,24 @@ python-mechanize:
   gentoo: [dev-python/mechanize]
   nixos: [pythonPackages.mechanize]
   ubuntu: [python-mechanize]
+python-mistune:
```

Comment:
Most of these are python 3 rules so I'd suggest that this be python3-mistune

For the older releases we can backfill pip for python3 (aka bionic and buster) 

And with that we'd call this python3-mistune

In general we're expecting all python 2 rules to eventually disappear as python2 is now strongly deprecated.